source/Calamari.Azure/Calamari.Azure.csproj

```diff
@@ -5,10 +5,13 @@
         <OutputType>Library</OutputType>
         <Authors>Octopus Deploy</Authors>
         <Copyright>Octopus Deploy Pty Ltd</Copyright>
-        <TargetFrameworks>net462;netstandard2.1</TargetFrameworks>
+        <RuntimeIdentifiers>win-x64;linux-x64;osx-x64;linux-arm;linux-arm64</RuntimeIdentifiers>
+        <LangVersion>8.0</LangVersion>
+        <TargetFrameworks>net462;net6.0</TargetFrameworks>
```

Comment:
Is there some consequences of doing this? I kinda did it blindly but the builds were working so I didn't worry about it too much 🤷

Reply:
I don't see an issue but @scme0 would be the guy with a better answer

Reply:
I *think* it should be ok as long as it's dependencies are happy with it.